testing/scenario_app/assemble_apk.sh

```diff
@@ -28,11 +28,15 @@ SRC_DIR="$(cd "$SCRIPT_DIR/../../.."; pwd -P)"
 export ANDROID_HOME="$SRC_DIR/third_party/android_tools/sdk"
 
 "$SRC_DIR/flutter/tools/gn" --unopt
-autoninja -C "$SRC_DIR/out/host_debug_unopt" sky_engine sky_services
+ninja -C "$SRC_DIR/out/host_debug_unopt" sky_engine sky_services -j 400
```

Comment:
autoninja doesn't work with local fuchsia RBE installations right now, and this task doesn't acutally do too much work anyway.